src/chainlit/frontend/src/components/organisms/slider.tsx

```diff
@@ -53,19 +53,21 @@ const StyledSlider = styled(Slider)(({ theme }) => {
 
 interface Props extends SliderProps {
   label: string;
+  tooltip?: string;
 }
 
-const _Slider = ({ label, ...sliderProps }: Props) => {
+const _Slider = ({ label, tooltip, ...sliderProps }: Props) => {
   return (
-    <Stack mr={1}>
+    <Box>
```

Comment:
I think the margin right was needed because the thumb (the circle you move on the slider) might not be rendered entirely if it is too close to the edge of the parent. Not sure though.

Reply:
I removed it since it wasn't aligned with the textinput boxes. I also think that if in a consumer of the component a margin is needed that consumer should be responsible to add it. I don't think you want to add it to the component itself since you will force all consumers to also have the margin by default. Thoughts?

I should have verified the current places where the slider is used, I will do that this evening and look if the margin is indeed needed.

Reply:
I think it makes sense, it should probably not be in the component directly.